.circleci/config.yml

```diff
@@ -353,15 +353,26 @@ commands:
           command: |
             wget -O ~/ff.tar.bz2 "https://download.mozilla.org/?product=firefox-nightly-latest-ssl&os=linux64&lang=en-US"
             tar -C ~ -xf ~/ff.tar.bz2
+      - run:
+          name: Add audio dependencies
+          command: |
+            # This should add and start PulseAudio's dummy mixer. It will warn
+            # that "This program is not intended to be run as root" but it can
+            # be ignored.
+            apt-get update -y; true
```

Comment:
Why `; true` there?  Does `apt-get update` fail for some reason?

Reply:
Yes, it's just in case it fails. The install is the important one, which if that fails then the test will definitely fail, so I was trying to reduce any flakiness.

Reply:
Removed.